core-bundle/src/Entity/WebauthnCredential.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Entity;
+
+use Contao\CoreBundle\Repository\WebauthnCredentialRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\Table;
+use Symfony\Component\Uid\Ulid;
+use Symfony\Component\Uid\Uuid;
+use Webauthn\PublicKeyCredentialSource;
+use Webauthn\TrustPath\TrustPath;
+
+#[Table(name: 'webauthn_credentials')]
+#[Entity(repositoryClass: WebauthnCredentialRepository::class)]
+class WebauthnCredential extends PublicKeyCredentialSource
+{
+    #[Column(type: Types::DATETIME_IMMUTABLE)]
+    public readonly \DateTimeImmutable $createdAt;
+
+    #[Id]
+    #[Column(unique: true)]
+    #[GeneratedValue(strategy: 'NONE')]
```

Comment:
Is this correct? Usually we have `'AUTO'`.

Reply:
This was taken straight from the [example](https://webauthn-doc.spomky-labs.com/symfony-bundle/entities-with-doctrine#the-doctrine-entity). This way you can have uuids as the primary key (generated in the constructor below).